src/common/RunParams.hpp

```diff
@@ -125,6 +125,8 @@ class RunParams {
 
   double getSizeFactor() const { return size_factor; }
 
+  size_t getMemoryAlignment() const { return memory_alignment; }
```

Comment:
Data alignment instead of Memory Alignment?

Reply:
That seems to be what we're using for RAJA so I'll make that change for consistency.